### DISCUSSION TITLE
Can’t find BlueRetro_dc_spiffs.bin in firmware updates for Retrotime DC adapter

### DISCUSSION THREAD

**Discussion by Notroker** [darthcloud/BlueRetro#1079] [Q&A]
Hi. As title states. I already mistakenly updated one of them with the wrong file and bricked it. This is a new one.  But I can’t find that file. Here is what I’m referring to:

https://8bitmods.com/dc-blueretro-controller-receiver-with-virtual-memory-for-dreamcast-smoke-black/

_Updating firmware via Web Config (OTA FW update):

Download the latest firmware from the BlueRetro GitHub page. Unzip the firmware file. Open a tab/window in your web browser and navigate to [Blueretro Update](https://blueretro.io/ota.html) Click “Connect BlueRetro” button. Select the “BlueRetro_dc_spiffs.bin” from the unzipped firmware folder. The “Live Output” will scroll through with the update process. On a PC, the update can take around 5 minutes. On Android, the update can take significantly longer.

This adapter is made possible by the amazing Blueretro open source project by Darthcloud: [GitHub](https://github.com/darthcloud/BlueRetro/)_

**Comment by darthcloud** [comment]
use BlueRetro_hw1_dreamcast.bin
